src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,25 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DataProcessingException;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static Manufacturer manufacturer;
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                    .getInstance(ManufacturerDao.class);
+        manufacturer = new Manufacturer("Tesla", "USA");
+        manufacturerDao.create(manufacturer);
+        manufacturer = manufacturerDao.get((long) 11)
```

Comment:
```suggestion
        manufacturer = manufacturerDao.create(manufacturer);
        manufacturer = manufacturerDao.get(manufacturer.getId())
```

then you don't need to follows the id

Reply:
ok